AnnService/src/Helper/VectorSetReaders/TxtReader.cpp

```diff
@@ -66,12 +66,12 @@ TxtVectorReader::LoadFile(const std::string& p_filePaths)
             return ErrorCode::FailedOpenFile;
         }
 
-        std::uint32_t fileTaskCount = 1;
+        std::uint32_t fileTaskCount = 0;
```

Comment:
I don't understand this change - isn't it reset either way?

Reply:
yes, so I just recover the previous code here.